client/src/lib/store/track.ts

```diff
@@ -16,6 +16,10 @@ export interface Application {
   dateApplied: string;
   applicationURL: string;
   applicationStatus: ApplicationStage;
+  currentScore?: string;
```

Comment:
Isn't the Application interface redundant with the JobCard props? Why not just import the JobCard props so you're only maintaining and using a single interface. It will make it consistent to only use a single type as the props of the JobCard grows larger

Reply:
I have the application from track default to pass in the rest of component props, instead of making a new one.